cred-manager/src/sourcecred/service/calculationQueue.service.ts

```diff
@@ -29,13 +29,18 @@ export class CalculationQueueService {
       });
       if (unhandledCalculation) {
         this.logger.debug("Handling unhandled calculation request.");
-        // update semaphore to unavailable
-        await this.prismaService.calculationSemaphore.update({
-          where: { id: semaphore.id },
-          data: { available: false }
-        })
-        // calculate cred scores
-        await this.sourceCredService.calculateCredScores(unhandledCalculation.team_id, unhandledCalculation.access_token, unhandledCalculation.email);
+        try {
+          // update semaphore to unavailable
+          await this.prismaService.calculationSemaphore.update({
+            where: { id: semaphore.id },
+            data: { available: false }
+          })
+          // calculate cred scores
+          await this.sourceCredService.calculateCredScores(unhandledCalculation.team_id, unhandledCalculation.access_token, unhandledCalculation.email);
+        } catch(e) {
+            this.logger.error(e);
+        }
```

Comment:
Why you added this try catch? AFAIK the error the user saw was when the calculation request was being deleted after the calculation was done

Reply:
The application did not recover,  if something broke on the calculation the con never goes into ready state again